rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/ScriptConsoleView.java

```diff
@@ -236,25 +245,37 @@ public void nodeUnselected() {
         this.label.show();
         this.nodeCanvas.hide();
         this.nodeUrl = null;
+        this.nodeSourceName = null;
+        this.nodeHostName = null;
     }
 
     public void nodeSelected(Node node) {
-        this.nodeLabel.setIcon(node.getNodeState().getIcon());
+        this.nodeLabel.setIcon(node.getIcon());
```

Comment:
in many places there is deleted getNodeState() method call. Why it is not needed any more?